jetty-webapp/src/test/java/org/eclipse/jetty/webapp/WebInfConfigurationTest.java

```diff
@@ -72,7 +73,7 @@ public void testFindAndFilterContainerPaths()
         context.setClassLoader(loader);
         config.findAndFilterContainerPaths(context);
         List<Resource> containerResources = context.getMetaData().getContainerResources();
-        assertEquals(1, containerResources.size());
+        assertEquals(1, containerResources.size(), () -> containerResources.stream().map(Resource::toString).collect(Collectors.joining(",", "[", "]")));
```

Comment:
I added these to troubleshoot the CI build/test failure on these.
CI is reporting more resources than expected.

Reply:
Fixed.  

Looks like the container include jar pattern ...

``` java
context.setAttribute(WebInfConfiguration.CONTAINER_JAR_PATTERN, ".*/jetty-util-[^/]*\\.jar$|.*/jetty-util/target/classes/");
```

... occasionally finds `jetty-util-ajax-9.4.34-SNAPSHOT.jar` as well.